plugins/node/opentelemetry-instrumentation-aws-sdk/test/sns.test.ts

```diff
@@ -58,7 +61,7 @@ describe('SNS', () => {
     } as AWS.SNS.Types.PublishResponse);
   });
 
-  describe('publish', () => {
+  describe('publish - v2', () => {
```

Comment:
nit: I think this current describe is already under the one named "SNS - v2", so we can omit the V2 here